Formula/yorkie.rb

```diff
@@ -0,0 +1,39 @@
+class Yorkie < Formula
+  desc "Document store for collaborative applications"
+  homepage "https://yorkie.dev/"
+  url "https://github.com/yorkie-team/yorkie.git",
+    tag:      "v0.2.6",
+    revision: "284ceddfaa6400ce6eaaffad4570393c729ebf81"
+  license "Apache-2.0"
+  head "https://github.com/yorkie-team/yorkie.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install Dir[buildpath/"bin/*"]
```

Comment:
```suggestion
    prefix.install "bin"
```
Though, is there no way to specify the prefix in the Makefile?

Reply:
Thanks, It looks good.